test/Microsoft.Health.Fhir.Ingest.Template.UnitTests/Microsoft.Health.Fhir.Ingest.Template.UnitTests.csproj

```diff
@@ -43,106 +43,10 @@
   </ItemGroup>
 
   <ItemGroup>
-    <None Update="TestInput\data_CodeValueFhirTemplate_CodeableConceptAndStringComponent.json">
+    <None Update="TestInput\*.json">
```

Comment:
Instead of hardcoding each file I used a regex. Please let me know if this isn't desired.